libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts

```diff
@@ -21,6 +21,7 @@ import {
 import { m } from './messages'
 import { YES, NO, FILE_SIZE_LIMIT } from '../constants'
 import { StatusTypes } from '../types'
+import { Address } from 'libs/api/schema/src/lib/schema'
```

Comment:
Thinks this import needs to be something like `@island.is/api/schema`, otherwise linting will fail